src/serverModuleGen.py

```diff
@@ -7,20 +7,22 @@ def getModules(filename):
         modules = configfile.readlines()
         ret = []
         for module in modules:
-            module = module.split("#")[0].split("/")[-1].strip()
+            module = module.split("#")[0].strip()
             if module:
                 ret.append(module)
         return ret
```

Comment:
if we actually like the code golf style we can also shorten this one a bit. With the added benefit of being first time I've ever actually used the walrus operator!
```python
return [m for module in modules if (m:=module.split("#")[0].strip())]
```

Reply:
and if we really want to get crazy, i think the entire file might be able to get down to just the below, but i haven't tested it
```python
import sys

if __name__ == "__main__":
    with open(sys.argv[1]) as configfile:
        get_modules = [m for module in configfile.readlines() if (m := module.split("#")[0].strip())]
    print(" ".join([f"{sys.argv[2]}/{mod}.chpl" if mod[0] != "/" else f"{mod}.chpl" for mod in get_modules]))
```